docs/source/quickstart.rst

```diff
@@ -100,12 +100,12 @@ For example::
 
     >>> from hyper import HTTPConnection
     >>> c = HTTPConnection('http2bin.org')
-    >>> first = c.request('GET', '/get')
-    >>> second = c.request('POST', '/post', body='key=value')
+    >>> first = c.request('GET', '/get', body='hello')
+    >>> second = c.request('POST', '/post', headers={'key':'value'})
```

Comment:
Can you swap these two around? It makes a bit more sense to have the body on the POST rather than the GET.